### ISSUE TITLE
Tabs don't work (click on them, but nothing happens)

### ISSUE THREAD

**Issue by vitalyliber** [expo/router#366]
### Summary

Hey! This is my second try to make tabs work. 
I see tabs, but clicking on them doesn't change anything.

@EvanBacon please, check my code + video 🙏 

I hope you have a simple answer to why it happened. 

### Minimal reproducible example

https://github.com/vitalyliber/expo-router-tabs-example

<video src="https://user-images.githubusercontent.com/8033090/224494180-74921d44-aaed-4f16-87bf-9d7a3903ee27.mp4"></video>

**Comment by mauricedoepke** [comment]
The Tabs are working fine. Your custom auth logic is constantly redirecting you to the home route though.

When I comment out the line: `router.replace("/home");` in your `context/auth/index.js` everything is working fine.

I found out by deleting stuff from your example app piece by piece, except for the tabs to narrow down the issue. When I deleted the auth logic it started working.
This method usually helps you finding most issues.
